themes/bootstrap3/templates/_ui/components/menu-button.phtml

```diff
@@ -0,0 +1,24 @@
+<?php
+  $wrapperTag = $this->wrapperTag ?? 'li';
+  $wrapperClass = $this->wrapperClass ?? 'li';
+
+  $toggleTag = $this->toggleTag ?? 'a';
+  $toggleAttrs = $this->toggleAttrs ?? [];
+?>
+
+<<?=$wrapperTag ?> class="<?=$wrapperClass ?> dropdown">
+  <<?=$toggleTag ?> href="#" class="dropdown-toggle" data-toggle="dropdown">
+    <?=$this->transEsc($this->toggleLabel)?>
+    <b class="caret"></b>
```

Comment:
Is this an icon? Even if not an icon, would a class name that's more functional (e.g. dropdown-indicator) make sense, in case we choose to style it with something other than a caret?

Reply:
This is a Bootstrap component piece, but we can probably replace it.

Reply:
I may misunderstand what you're after, but I'd opt for removing the empty \<b>\-tag. It makes sense but I guess it's more of a gut feeling thing. 
What I'd prefer would be something resulting in an item more in the vein of
\<a 
     class="<?=$dropdown-indicator-class ?> dropdown-toggle" 
     href="#" 
     role="button" 
     id="..."
     data-toggle="dropdown" 
     aria-haspopup="true" 
     aria-expanded="false">\
    Dropdown menu text AND/OR Icon + sr-only or arial-label or arial-labeledby
\</a>

<?=$dropdown-indicator-class ?> we could use if we want it to be a button, say "btn btn-secondary" -- maybe even more specific such as <?=$header-dropdown-indicator-class ?> and <?=$searchbox-dropdown-indicator-class ?> 

An id would be usefull too for using in aria-labeledby for the dropdown-menu item itself.

Should it also be possible to turn this into a button as part of a button group as for bulk actions? So we get something like
\<button 
   type="button" 
   class="\<?=$dropdown-indicator-class ?> dropdown-toggle" 
   data-toggle="dropdown" 
   aria-haspopup="true" 
   aria-expanded="false">
    Dropdown button text AND/OR Icon + sr-only or arial-label or arial-labeledby
\</button>